backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/interim/invoice/InterimInvoiceCandidateService.java

```diff
@@ -188,10 +193,10 @@ private DocTypeId getInterimInvoiceDocType()
 		if (interimInvoiceDocType == null)
 		{
 			interimInvoiceDocType = docTypeDAO.getDocTypeId(DocTypeQuery.builder()
-					.adClientId(Env.getAD_Client_ID())
-					.docBaseType(DocBaseType.APInvoice)
-					.docSubType(X_C_DocType.DOCSUBTYPE_DownPayment)
-					.build());
+																	.adClientId(Env.getAD_Client_ID())
```

Comment:
i know that it was before but using Env.getAD_Client_ID() is totally wrong!
consider:
* using ClientId.METASFRESH
* provide clientId as param